transports/tcp/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## 0.41.0
```

Comment:
Just realized that `0.41.0` is already released so this will need to be `0.42.0`, sorry!

Plus, you'll also need to bump the version in `transports/tcp/Cargo.toml` and the reference in `[workspace.dependencies]` in the root `Cargo.toml`!

Thank you :)

Reply:
The `0.41.0` release was an "empty" release in that it only bumped dependencies and didn't have any relevant changelog entries. So leave the `0.41.0` section as is and add a new `0.42.0` on top of it please.